packages/cli/src/utils/exec.ts

```diff
@@ -0,0 +1,52 @@
+import { spawn, SpawnOptions } from 'child_process'
+
+export function execCmd(
```

Comment:
at this point... having so many instances of this fn, shouldn't we move it to celo-utils?

Reply:
We should, but I'm not sure I want to be the person transforming the codebase, nor prescribe which version should be the one to rule them all